routes/my_api.js

```diff
@@ -36,9 +37,12 @@ const Config = require('../config');
 
 const CloudSync = require('./cloud-sync');
 const MyConversation = require('./my_conversation');
+const db = require('../util/db');
 
 var router = express.Router();
 
+router.use(urlencoded({extended:false}));
```

Comment:
You don't need this, in fact, you should not have it, because it's already installed by frontend.js